src/System.Xml.XmlSerializer/tests/System.Xml.XmlSerializer.Tests.csproj

```diff
@@ -17,7 +17,9 @@
     <Compile Include="XmlSerializerTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(CommonTestPath)\System\Console.cs" />
+    <Compile Include="$(CommonTestPath)\System\Console.cs">
+      <Link>Common\System\Console.cs</Link>
```

Comment:
The comment in that file says:
```C#
    // A dummy class for passing through System.Console calls until the System.Console contract is available.
    // To-do: Remove this workaround when possible.
```
and then implements WriteLine to call Debug.WriteLine.